packages/typography/src/SharedLink/SharedLink.styles.ts

```diff
@@ -0,0 +1,89 @@
+import { css } from '@leafygreen-ui/emotion';
```

Comment:
Would it make more sense to have a directory structure like below? That way we don't have a folder with only one styles file.

```
src
|- Link
|--- index.ts
|--- shared.styles.ts
|--- Link
|------ Link.styles.ts
|------ Link.tsx
|------ ...etc
|--- BackLink
|------ BackLink.styles.ts
|------ BackLink.tsx
|------ ...etc
```

Reply:
yeah that makes more sense